keyboards/redox_w/readme.md

```diff
@@ -0,0 +1,34 @@
+Redox wireless
+=======
+
+<p align="center">
+<img src="https://github.com/mattdibi/redox-keyboard/raw/master/img/redox-logo.png" alt="Redox logo" width="600"/>
+</p>
+
+**Redox**: the **R**educed **E**rgo**dox** project. More information and building instruction [here](https://github.com/mattdibi/redox-keyboard).
+
+A wireless version of the Redox keyboard.
+
+- Keyboard Maintainer: [@mattdibi](https://github.com/mattdibi)
```

Comment:
Lines 12 and 13 both need two spaces added at the end for this part to be rendered properly when viewed.

Reply:
Do you mean something like this?

```diff
diff --git a/keyboards/redox_w/readme.md b/keyboards/redox_w/readme.md
index 87924ad2d..a5536330b 100644
--- a/keyboards/redox_w/readme.md
+++ b/keyboards/redox_w/readme.md
@@ -10,7 +10,11 @@ Redox wireless
 A wireless version of the Redox keyboard.
 
 - Keyboard Maintainer: [@mattdibi](https://github.com/mattdibi)
+
+
 - Hardware Supported: Redox-w rev1.0W PCB
+
+
 - Hardware Availability: Falbatech
 
 Make example for this keyboard (after setting up your build environment):
```